integration/substrate/runtime_errors.spec.ts

```diff
@@ -4,103 +4,68 @@ import { ContractPromise } from "@polkadot/api-contract";
 
 describe('Deploy runtime_errors.sol and test the debug buffer', () => {
     it('logs errors to the debug buffer', async function () {
-
         let conn = await createConnection();
         const alice = aliceKeypair();
-
-
         let deployed_contract = await deploy(
             conn,
             alice,
             "RuntimeErrors.contract",
             BigInt(0)
         );
-
         let contract = new ContractPromise(
             conn,
             deployed_contract.abi,
             deployed_contract.address
         );
-
         let child_contract = await deploy(conn, alice, 'child_runtime_errors.contract', BigInt(0));
 
         let res = await debug_buffer(conn, contract, `get_storage_bytes`, [])
```

Comment:
In Typescript, we use `let` for mutable variable and `constant` for those whose value does not change.